src/components/TabMenu/index.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Item from './Item';
+import style from './style.css';
+
+const TabMenu = ({ align, className, ...props }) => {
+  const tabMenuClass = classNames(
+    style.TabMenu,
+    className,
+    style[`${align}Align`]
```

Comment:
Super minor: `styled-system` has a utility for flex styles, if we still feel ok about that pattern. One `flex` util in classNames would let the consumer apply a whole host of flex styles.

Even if we don't double down on styled-system, what would you think of using `justifyContent` instead of `align` as the prop type? It will be more transparent to the end user what they're changing.

Reply:
I don't love it, simply because at that point you might as well put all this stuff in a passed style prop. The point of SimpleTabMenu is to be as simple and human-readable as possible... but i can be swayed i guess